lib/compute/agent-node-config.ts

```diff
@@ -6,6 +6,8 @@
  * compatible open source license.
  */
 
+/* eslint-disable max-len */
+/* eslint-disable eqeqeq */
```

Comment:
Can you fix this rather than disabling, the imports can be placed on 2 lines.

Reply:
@gaiksaya `eqeqeq` we can limit for mac as if you see the value is generic
```
value: '/usr/local/opt/python@3.7/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/Cellar/python@3.7/3.7.13_1/Frameworks/Python.framework/Versions/3.7/bin',
```
let me try to add specific to those lines.